server/zanata-frontend/pom.xml

```diff
@@ -46,46 +45,52 @@
               the editor at http://<server>/app -->
     <editor.bundle.destination>${bundle.dest}/app</editor.bundle.destination>
 
-    <!-- Directory that contains the actual built frontend app (index, script and styles) -->
-    <frontend.built.artifact.directory>${frontend.build.directory}/dist</frontend.built.artifact.directory>
-
-    <!-- Directory that contains the actual built editor app (index, script and styles) -->
-    <editor.built.artifact.directory>${editor.build.directory}/app/dist</editor.built.artifact.directory>
   </properties>
 
   <dependencies>
   </dependencies>
 
   <build>
-    <!-- Copy all the source files to the build directory.
-         This is done to keep the original source directory clean, because the
-         build will download dependencies and places built artifacts in the
-         same directory as the source files.
-         This happens in phase process-resources. -->
-    <resources>
-      <resource>
-        <!-- Do not substitute maven variables like ${variablename} in the
-             contents of these files. -->
-        <filtering>false</filtering>
-        <!-- Path from pom to the source files that will be used. -->
-        <directory>src</directory>
-        <excludes>
-          <exclude>**/node_modules/</exclude>
-        </excludes>
-        <!-- Location to copy source files to. -->
-        <targetPath>${project.build.directory}/src</targetPath>
-      </resource>
-    </resources>
     <plugins>
-      <!-- Installs correct version of node and npm locally. -->
+      <plugin>
+        <groupId>org.codehaus.gmavenplus</groupId>
+        <artifactId>gmavenplus-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>default</id>
+            <phase>initialize</phase>
+            <goals>
+              <goal>execute</goal>
+            </goals>
+          </execution>
+        </executions>
+        <!-- check for ignore-scripts = true in .npmrc, which breaks npm -->
+        <configuration>
+          <scripts>
+            <script>
+              def ignoreScripts = new File(System.getProperty('user.home'), '.npmrc').text.readLines().any{it.matches(/\s*ignore-scripts\s*=\s*true\s*/)}
```

Comment:
@djansen-redhat Yep, I had intended to go back and handle that case and then forgot about it.

@huangp Another small change to review, please.